### DISCUSSION TITLE
Table names with a prefix underscore..

### DISCUSSION THREAD

**Discussion by vpray-uk** [go-jet/jet#357] [Q&A] [answered]
The Jet generator names the generated Go files by following the table names. This creates errors as the files with a prefix underscore are ignored. My current workaround is to manually rename those files but want to check if there's any better solution that I don't know of?

**Comment by houtn11** [comment, marked as answer]
You can customize the generator: https://github.com/go-jet/jet/wiki/Generator#generator-customization.
